dlib/media/ffmpeg_abstract.h

```diff
@@ -29,6 +29,22 @@ namespace dlib
                 - Returns a string description of a channel layout, where layout is e.g. AV_CH_LAYOUT_STEREO
         !*/
 
+// ---------------------------------------------------------------------------------------------------
+
+        dlib::logger& logger_internal();
+        /*!
+            ensures
+                - Returns a global logger used by the internal ffmpeg libraries. 
+                - You may set the logging level using .set_level() to supress or enable certain logs.
+        !*/
+
+        dlib::logger& logger_dlib();
+        /*!
+            ensures
+                - Returns a global logger used by dlib's ffmpeg wrappers.
+                - You may set the logging level using .set_level() to supress or enable certain logs.
+        !*/
+
```

Comment:
How about naming the loggers, either:
- `logger_ffmpeg`
- `logger_dlib`

or

- `logger_internal`
- `logger_wrapper`

Just to be a bit more consistent? Maybe a bit more biased towards the second option.

Reply:
Yeah maybe the second is better.